xm-commons-domain-event/src/main/java/com/icthh/xm/commons/domain/event/config/XmDomainEventConfiguration.java

```diff
@@ -0,0 +1,103 @@
+package com.icthh.xm.commons.domain.event.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.icthh.xm.commons.config.client.api.RefreshableConfiguration;
+import com.icthh.xm.commons.migration.db.liquibase.LiquibaseRunner;
+import com.icthh.xm.commons.tenant.TenantContextHolder;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.AntPathMatcher;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+@Slf4j
+@Component
+public class XmDomainEventConfiguration implements RefreshableConfiguration {
```

Comment:
If service is not using DB, this component will fail to start. I propose to make it conditional or remove @Component. Target service will create in in config phase if needed

Reply:
@major13ua we will discuss this tommorow and get back to you

Reply:
seems to be resolved already. @major13ua pls approve/comment